state.go

```diff
@@ -1023,7 +1038,7 @@ func (s *State) OnInterface(se *Session, i interface{}) (err error) {
 		}
 	case *GuildEmojisUpdate:
 		if s.TrackEmojis {
-			err = s.EmojisAdd(t.GuildID, t.Emojis)
+			err = s.EmojisUpdate(t.GuildID, t.Emojis)
```

Comment:
Since the implementation is very simple and short, I think it would be better to implement it inline (see `GuildMembersChunk` and `GuildMemberRemove` as examples)